aerogearcatalog/sync-app-apb
# Aerogear 8260

# What this does

Adds a bind and unbind options

# What might be missing

The current information in the binding is rather minimal, but that all can be added now that binding/unbinding works.

# How to test

You should be able to run `apb bundle prepare` then do a docker build and a docker push to whatever repository your broker is looking at.  You can then deploy the service and bind/unbind from MDC per normal.
